node/src/accountant/scanners/mod.rs

```diff
@@ -1447,525 +1076,104 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(expected = "Complete me with GH-605")]
-    fn retry_payable_scanner_panics_in_case_no_qualified_payable_is_found() {
-        let consuming_wallet = make_paying_wallet(b"consuming wallet");
-        let now = SystemTime::now();
-        let (_, unqualified_payable_accounts, _) =
-            make_qualified_and_unqualified_payables(now, &PaymentThresholds::default());
-        let payable_dao =
-            PayableDaoMock::new().non_pending_payables_result(unqualified_payable_accounts);
-        let mut subject = PayableScannerBuilder::new()
-            .payable_dao(payable_dao)
-            .build();
-
-        let _ = Scanners::start_correct_payable_scanner::<ScanForRetryPayables>(
-            &mut subject,
-            &consuming_wallet,
-            now,
-            None,
-            &Logger::new("test"),
-        );
-    }
-
-    #[test]
-    fn payable_scanner_handles_sent_payable_message() {
+    fn finish_payable_scan_keeps_the_aware_of_unresolved_pending_payable_flag_as_false_in_case_of_err(
```

Comment:
I don't wanna bully you but it might be a good idea factor out the body of this test and make two diff tests to cover both payable modes. I know that they are almost identical and there probably isn't anything what could make behave one differently than the other but it's safer if have it cover even for potential future modifications of the code...